pkg/state/state_whereabouts_cni.go

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
Anything changed here from previous round of updates ? (github shows me this file as updated)